### ISSUE TITLE
Issues with cleaning recent documents and System Hibernation File

### ISSUE THREAD

**Issue by vishakhg98** [mirinsoft/CleanmgrPlus#8]
1)Cleaning recent documents unpins the pinned folders in quick access bar.
2)And cleaning System Hibernation disables the fast boot option(cause hibernation turns off) and BSOD happens frequently.
Enabled the hibernation using following command in command prompt and everything's okay now
powercfg -hibernate on

**Comment by Oleekae** [comment]
Yes, the same in my case for the point 1. Please fix this behaviour. 

is it possible to exclude any file in the temporary files so that it is not deleted, if so how? TIA

**Comment by Belim** [comment]
1. Fixed in build https://github.com/mirinsoft/CleanmgrPlus/releases/tag/1.33.1063
2. But that's exactly what the function does. Otherwise, I would have to take the function out completely

**Comment by Oleekae** [comment]
1- Great thanks Belim for fix the point 1.
2- Yes, but in this case, although it seems a lie, I have a program that if i delete its temporary i have to activate it again. If you could implement an option to be able to exclude "xx" file/s as a white list. Anyway thank you very much for the development of this nice project.
